lib/ex_aws/sts.ex

```diff
@@ -0,0 +1,44 @@
+defmodule ExAws.STS do
+
+  @moduledoc """
+  Operations on AWS STS
+
+  http://docs.aws.amazon.com/STS/latest/APIReference/API_Operations.html
+  """
+
+  @type policy :: %{
+    binary => :all
+  }
+
+  @type get_federation_token_params :: %{
+    :duration => pos_integer(),
+    :name => binary,
+    :policy => policy
+  }
+
+  @doc "Get Federation Token"
+  @spec get_federation_token(params :: get_federation_token_params) :: ExAws.Operation.Query.t
+  def get_federation_token(%{duration: duration, name: name, policy: policy}) do
```

Comment:
So to be clear, in this case duration and policy are optional, so the signature should be:

`get_federation_token(name, [duration: duration, policy: policy])`